temp/legacy-code/src/main/java/com/ivy/legacy/ui/component/transaction/TransactionCard.kt

```diff
@@ -457,19 +460,45 @@ private fun TransactionBadge(
 @Composable
 private fun TransferHeader(
     accounts: List<Account>,
-    transaction: Transaction
+    transaction: Transaction,
+    shouldShowAccountSpecificColorInTransactions: Boolean
 ) {
+    val account = accounts.find { transaction.accountId == it.id }
```

Comment:
This must be:
- executed only if the `shouldShowAccountSpecificColorInTransactions` feature flag is enabled
- remembered using `remember(transactions, accounts) {}`

Reply:
I don't think so because i will have to show the account name any way

Reply:
Got it, the  just remember the O(n) account search computation